bats/tests/containers/factory-reset.bats

```diff
@@ -115,19 +129,19 @@ check_installation() {
                 fi
             fi
         done
-    fi
 
-    for profile in "${env_profiles[@]}"; do
-        echo "$assert that $profile does not add ~/.rd/bin to the PATH"
-        # cshrc: setenv PATH "/Users/jan/.rd/bin"\:"$PATH"
-        # posix: export PATH="/Users/jan/.rd/bin:$PATH"
-        run grep "PATH.\"$HOME/.rd/bin" "$profile"
-        ${assert}_failure
-    done
+        for profile in "${env_profiles[@]}"; do
+            echo "$assert that $profile does not add ~/.rd/bin to the PATH"
+            # cshrc: setenv PATH "/Users/jan/.rd/bin"\:"$PATH"
+            # posix: export PATH="/Users/jan/.rd/bin:$PATH"
+            run grep "PATH.\"$HOME/.rd/bin" "$profile"
+            ${assert}_failure
+        done
+    fi
 
     # Check if the rancher-desktop docker context has been removed
-    if using_docker; then
-        if is_unix; then
+    if is_unix; then
```

Comment:
This is continuing the `is_unix` block, so can be merged into the previous block.